frontend/wallet/src/render/Preload.re

```diff
@@ -24,10 +24,17 @@ let isFaker =
   Js.Dict.get(Bindings.ChildProcess.Process.env, "GRAPHQL_BACKEND")
   == Some("faker");
 
+let getTranslation = name =>
+  Bindings.Fs.readFileSync(
+    "./src/render/i18n/translations/" ++ name ++ ".json",
```

Comment:
My only concern is that our bundler might not include these files in the build output. A better place for the translation JSON files to live might be under `public/translations/` since all the files in that folder will automatically copied into the destination. Might be worth just running `yarn dist` and running that build to double check.

Reply:
I’ll test this!

Are there any tests running against the prod build yet? Was thinking it would catch this error since loading a missing file would throw an error

Reply:
@Schmavery can we pair again sometime to finish this up?

1. Unable to commit now, receiving this error. Unsure why that particular exe is missing.
```
INFO Not using Docker
find: _build/default: No such file or directory
find: _build/default: No such file or directory
find: _build/default: No such file or directory
dune exec --profile=dev src/app/reformat/reformat.exe -- -path . -check
make: dune: No such file or directory
make: *** [check-format] Error 1
** REFUSING TO COMMIT: "$ make check-format" failed
```

2. `yarn dist` leaves a long error stack trace.
```
Can't locate Mac/Memory.pm in @INC (you may need to install the Mac::Memory module)...
```

Reply:
@dangdennis 

1) that's a post commit hook that checks the native code (surprised you never ran into it before). If you have opam+dune installed, you can `eval $(opam env)` and then it should work, otherwise, you can skip the hooks with `git commit --no-verify`

2) ouch, that's an ugly error. @figitaki did you see this when running dist build lately? It could be that upgrading to `"electron-builder": "^21.2.0"` (from [here](https://github.com/CodaProtocol/coda/pull/4084)) might help?

Reply:
Cool bypassed the git hook. 

Did a test upgrade to electron-builder to 21.2.0. I'm no longer getting that scary stack trace. But what's `build` doing in this script: 
`"dist": "yarn run build && yarn run pack && build"`

How does electron-builder actually get run in your setup?

Edit: 
`"dist": "yarn run build && yarn run pack && electron-builder build"` produced a running .dmg!